roles/openshift_health_checker/openshift_checks/logging/logging.py

```diff
@@ -8,11 +8,33 @@
 from openshift_checks import OpenShiftCheck, OpenShiftCheckException
 
 
+class MissingComponentPods(OpenShiftCheckException):
+    """Raised when a component has no pods in the namespace."""
+    pass
+
+
+class CouldNotUseOc(OpenShiftCheckException):
+    """Raised when ocutil has a failure running oc."""
+    pass
+
+
+class LoggingErrorList(OpenShiftCheckException):
```

Comment:
Perhaps this doesn't need to have "Logging" in the name? It could as well sit next to "OpenShiftCheckException" and be  an alternative to it? `class OpenShiftCheckExceptionList(OpenShiftCheckException)`?